saved-to-the-uttermost/updated/06-chapter-5.adoc

```diff
@@ -419,6 +423,7 @@ footnote:[2 Corinthians 5:18-19]
 From here it is inferred that Christ fully perfected
 the work of reconciliation while He was on earth.
 
+[.discourse-part]
```

Comment:
I see that you're doing tons of discourse-part's without the manual italics here, so I'm guessing the spot I noted above was just an oversight.